src/main/java/edu/wisc/my/portlets/feedback/web/FeedbackFormController.java

```diff
@@ -84,13 +84,15 @@ protected Object formBackingObject(PortletRequest request) throws Exception {
     	final PortletPreferences preferences = request.getPreferences();
     	
     	if(preferences !=null){
-    		request.setAttribute("chatLink",(preferences.getValue("chatLink", null)));
-    		request.setAttribute("callLink",(preferences.getValue("callLink", null)));
-    		request.setAttribute("howToLink",(preferences.getValue("howToLink", null)));
+    		request.setAttribute("chatLink",(preferences.getValue("chatLink", "null")));
```

Comment:
I wouldn't worry too much about it. Most of the value was in being sure the change was deliberate rather than accidental. Clean history is nice, but the marginal value is slim.